### ISSUE TITLE
corrupt daemon file?

### ISSUE THREAD

**Issue by martonisummit** [Fjuxx/pimatic-otgw#2]
I have been struggling with the daemon file (all else is in working order, I am not using pimatic for now)
The  script seems to have become corrupt?

Due to spurious spaces on various lines....

**Comment by user34756361233** [comment]
That is correct. Repaired the file (executable in my case is otmonitor-x64). See attached
[otmonitor init.d script.txt](https://github.com/Fjuxx/pimatic-otgw/files/67764/otmonitor.init.d.script.txt)
